addons/web/static/tests/mobile/views/view_dialog/select_create_dialog_tests.js

```diff
@@ -0,0 +1,110 @@
+/** @odoo-module */
+
+import { click, getFixture } from "@web/../tests/helpers/utils";
+import { makeView, setupViewRegistries } from "@web/../tests/views/helpers";
+
+QUnit.module("ViewDialogs", (hooks) => {
+    let serverData;
+    let target;
+
+    const PRODUCT_PRODUCT = "product.product";
```

Comment:
would get rid of the constants and rename this into `product` s.t. we can use the simple dot notation to access array's elements